layouts/shortcodes/docs/instrumentation/manual-intro.md

```diff
@@ -0,0 +1,16 @@
+[Manual instrumentation](/docs/concepts/instrumentation/manual/) is the act
+of adding observability code to your application yourself.
+
+If you're instrumenting an application, you need to use the OpenTelemetry SDK
+for your language. You'll then use the SDK to initialize OpenTelemetry and the
+API to instrument your code. This will emit telemetry from your app and any
```

Comment:
I'm ok with your use of "app" here, but let's be consistent and replace "application" in the rest of this diff hunk by "app".

Reply:
fixed